src/skin/options.html

```diff
@@ -183,6 +183,18 @@ <h1><span class="i18n_options_title"></span></h1>
 
   <div id="tab-manage-data">
 
+    <div class="btn-silo">
+      <h2 id="sync-heading"><span class="i18n_sync_whitelist"></span></h2>
+      <div id="upload">
+        <h3><span class="i18n_upload_whitelist"></span></h3>
+        <button id="uploadWhitelist"><span class="i18n_upload"></span></button>
+      </div>
+      <div id="download">
+        <h3><span class="i18n_download_whitelist"></span></h3>
+        <button id="downloadWhitelist"><span class="i18n_download"></span></button>
```

Comment:
Do we need both "download" and "merge"? I think just "merge" should be enough. We may not want to call it "merge" either ... Importing user data also merges disabled site lists. So, get rid of "download" and rename "merge" to "download"?

Reply:
removed merge, download merges lists